### ISSUE TITLE
How to make dots different sizes in scatterplot

### ISSUE THREAD

**Issue by lainglo** [almende/vis#1198]
Hi,

I am trying to make a 2-D scatterplot with the  radius of the dots dependant on another property of each observation. I attempted to do this by adding a new property to each item in my DataSet in addition to the required properties. 
ie:
```javascript
items = new vis.DataSet([{x: '2014-06-13', y: 30, group: 0, budget: 10}, {x: '2014-08-17', y: 35, group: 1, budget: 20}, ... ,]);
```
I have also written a function in the drawPoints property of the configuration options, but I am not able to access my newly defined properties for the item within this function:

```javascript
var scatter_options = {
                  stack: true,
                  sort: false,
                 .....etc,
                  drawPoints: {
                        enabled: true,
                        style: 'circle',
                        onRender: (item, group, grap2d) {
                                console.log(item.budget); // console returns: "undefined"
                            return {
                                     size: item.budget
                                        };
                              };, //on Render function
                      //defaultGroup: 'Scatterplot',
                        }, //drawPoints
                }; 
```

Is there any way to accomplish this?

It seems like this is easy to do in graph3D. Would it maybe be easier to convert my data set to graph3d format and then just change the graph options so that the chart view is fixed to 2 dimensions?

BTW thanks for making such great software - I've been using the visjs for the past month or so now.

Regards

**Comment by AlexDM0** [comment]
Hi,

This is done using css, look at the className field and the examples on our website.

Regards

**Comment by AlexDM0** [comment]
Sorry, I misread your issue. At the moment this is not possible. We'll keep it in mind for when we refactor the graph2d.

Regards

**Comment by ludost** [comment]
I'm not 100% sure, but this is probably fixed in 4.11. The original data is now propagated all the way to the onRender function, so this should work. I hope to be able to test is later this week.

**Comment by ludost** [comment]
ok, I've checked. This has been fixed in release 4.11.
